src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,85 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final String REGEX = "[\\W]";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] result = readFromFile(fromFileName);
+        writeToFile(toFileName, dataProcessing(result));
+
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        List<String> strings;
 
+        try {
+            strings = Files.readAllLines(Path.of(fromFileName));
+            if (strings.toString().equals("[]")) {
+                return new String[0];
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open file! " + fromFileName, e);
+        }
+
+        for (String line : strings.toString()
+                .substring(1, strings.toString().length() - 1)
+                .split("\n")) {
+            stringBuilder.append(line)
+                    .append(" ");
+
+        }
+        return stringBuilder.toString().split(" ");
```

Comment:
```suggestion
         return strings.toArray(new String[0]);
```
this way you won't need all this code above